packages/sku/lib/configure.js

```diff
@@ -124,10 +124,7 @@ module.exports = async ({ isPostInit } = {}) => {
     const lessFileGlobResults = await Promise.all(
       paths.src
         .filter((srcPath) => fs.statSync(srcPath).isDirectory())
```

Comment:
This comment would be useful as a code comment https://github.com/seek-oss/sku/pull/802#discussion_r1198393128